roadmap/4_governance_nns/upcoming/1_nns_large_number_of_neurons.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Support large number of neurons
+links:
+  Forum Post:
+  Proposal:
+eta: Q3 23
+is_community: false
+---
+The number of neurons have beem steadily increasing during the last year and is reaching 200,000. Today all neurons, disbursed or not, are kept forever in heap memory. As this memory is limited, so are the number of neurons that can be created.
```

Comment:
```suggestion
The number of neurons have beem steadily increasing during the last year. Today all neurons, disbursed or not, are kept forever in heap memory. As this memory is limited, so are the number of neurons that can be created. This feature aims to redesign this so that new NNS neurons can be created going forward. 
```